MIGRATE.md

```diff
@@ -52,14 +52,18 @@ server or note its path if the same server will be used.
 
 1. Make sure your rmt installation is up-to-date. `rmt-data-import` is available in RMT versions >= 1.0.0
 2. Unpack the tarball containing SMT data to some directory, e.g. `/root/smt-data`
-3. If you chose to export SMT's SSL certificates, copy the SMT CA certificate to `/usr/share/rmt/ssl/`:
+3. If you chose to export SMT's SSL certificates, copy the SMT CA private key to `/usr/share/rmt/ssl/`:
+    ```
+    cp /root/smt-data/ssl/cacert.key /usr/share/rmt/ssl/rmt-ca.key
+    ```
+4. If you chose to export SMT's SSL certificates, copy the SMT CA certificate to `/usr/share/rmt/ssl/`:
```

Comment:
Just collapse this and previous item together?

Reply:
done

Reply:
Maybe collapse the ` ``` ` blocks too?

![image](https://user-images.githubusercontent.com/1793699/42216604-a356f472-7ec2-11e8-85ad-b44218dd0941.png)